clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1081,7 +1081,7 @@ class SyclKernelFieldHandlerBase {
   virtual bool enterField(const CXXRecordDecl *, FieldDecl *) { return true; }
   virtual bool leaveField(const CXXRecordDecl *, FieldDecl *) { return true; }
   virtual bool enterArray() { return true; }
-  virtual bool nextElement(QualType) { return true; }
+  virtual bool nextElement(QualType, int64_t) { return true; }
```

Comment:
nit (and only if doing something else to this patch), likely want this to be unsigned.

Reply:
Oooh I missed this comment and I JUST pushed a new patch fixing clang-format. I'll wait to see if there are any other other review comments and add this with that.

Reply:
Done